scarlet2/__init__.py

```diff
@@ -16,3 +16,4 @@ class Scenery:
 from .source import Component, DustComponent, Source, PointSource
 from .spectrum import Spectrum, ArraySpectrum, StaticArraySpectrum, TransientArraySpectrum
 from .wavelets import Starlet
+from .io import model_to_h5, model_from_h5
```

Comment:
Please remove this line. I don't want to force people to install hdf5 if they don't need this form of serialization